inductiva/molecules/protein_solvation/README.md

```diff
@@ -57,24 +55,30 @@ The last code line downloads the files necessary to render and post-process our
 The simulation output folder contains trajectory data spanning the simulation duration and the protein's topology. These files are necessary to generate an interactive trajectory visualization using [NGLview](https://github.com/nglviewer/nglview). Users have the flexibility to select the protein's representation type, including options to display or hide the backbone, and they can also specify the criteria for visualization selection. You can refer to the [documentation](https://nglviewer.org/ngl/api/manual/usage/selection-language.html) for guidance on creating custom selections according to your preferences.
 
 ```python
-output = task.get_output()
-output.render_interactive(representation="ribbon", add_backbone=True, selection="protein")
+output.render_interactive(representation="ball+stick", add_backbone=True, selection="protein")
  ```
-![protein_solvation](https://github.com/inductiva/inductiva/assets/114397668/e9dfe7d9-9c2b-4be4-90c3-0d68a2a7b9fd)
+<p align="center">
+  <img src="https://github.com/inductiva/inductiva/assets/114397668/4d2265c1-dfe7-435f-936d-f4d607e29a04" alt="Centered Image" width="350" height="250">
```

Comment:
This visual is weird, since it has a white contour around everything. Is it possible to remove that?

Reply:
I removed the background, maybe that's it but in mine looks fine

Reply:
There might be a difference between night mode and day mode.

Reply:
Ok, I will put the one with the background then!